modules/ROOT/partials/configuration/advanced-typography.adoc

```diff
@@ -0,0 +1,247 @@
+[[content_langs]]
+== `+content_langs+`
+
+This option sets which languages are available for applying language-specific typography rules.
+
+A user can then select text and select from the available languages. That language’s language-specific typography rules are then applied to the selected text.
+
+*Type:* `+Object+`
+
+=== Example: Using `content_langs`
+
+[source,js,subs="attributes+"]
+----
+tinymce.init({
+  selector: 'textarea', // change this value according to your html
+  plugins: '{plugincode}',
+  toolbar: '{plugincode} language',
+  typography_default_lang: [ "en-US", "de", "es" ], // Required to set specific typography language rules.
+  content_langs: [
+    { title: 'English', code: 'en' },
+    { title: 'Spanish', code: 'es' },
+    { title: 'German', code: 'de' }
+    ],
+});
+----
+
+=== Currently supported languages
+
+[cols="1,1"]
+|===
+|Supported language |Language code 
+
+|Belarusian
+|be
+
+|Bulgarian
+|bg
+
+|Catalan
+|ca
+
+|Czech
+|cs
+
+|Danish
+|da
+
+|Dutch
+|nl
+
+|English (UK)
+|en-GB
+
+|English (US)
+|en-US
+
+|Esperanto
+|eo
+
+|Estonian
+|et
+
+|Finnish
+|fi
+
+|German
+|de
+
+|Greek
+|el
+
+|Hungarian
+|hu
+
+|Irish
+|ga
+
+|Italian
+|it
+
+|Latvian
+|lv
+
+|Norwegian
+|no
+
+|Polish
+|pl
+
+|Romanian
+|ro
+
+|Russian
+|ru
+
+|Servian
+|sr
+
+|Slovak
+|sk
+
+|Slovenian
+|sl
+
+|Spanish
+|es
+
+|Swedish
+|sv
+
+|Turkish
+|tr
+
+|Ukraninan
+|uk
+
+|===
+
+////
+[[typography_custom_rules]]
```

Comment:
- [ ] remove all code that isn't ready.

Reply:
@ephox-mogran Removed typography_custom_rules template for now.
@brian-forte ref only